gui/resources.xml

```diff
@@ -763,27 +784,35 @@ Vocabulary
       <!-- {{{ inktool points related actions -->
       <child>
         <object class="GtkAction" id="InsertCurrentNode">
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Insert a node back of the currently selected node.</property>
+          <property name="label" translatable="yes" context="Accel Editor (labels)">Insert Inking Node</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Insert a inking node on the back of the currently selected node.</property>
           <signal name="activate" handler="insert_current_node_cb"/>
         </object>
+        <accelerator key="Insert"/>
```

Comment:
:-1: This is only relevant in Inking mode, and it doesn't have a menu entry, so it doesn't need a shortcut. let's leave it open till we add a generic "insert a thing on the current layer" action that activates this one if you're currently inking.